policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -14,27 +14,32 @@ use linkerd_policy_controller_core::{
 };
 use linkerd_policy_controller_k8s_api::{
     gateway::{self as k8s_gateway_api, ParentReference},
-    policy as linkerd_k8s_api, ResourceExt, Service,
+    policy as linkerd_k8s_api, Pod, ResourceExt, Service,
 };
 use parking_lot::RwLock;
-use std::{hash::Hash, net::IpAddr, num::NonZeroU16, sync::Arc, time};
+use std::{hash::Hash, net::IpAddr, num::NonZeroU16, str::FromStr, sync::Arc, time};
 use tokio::sync::watch;
 
+pub mod grpc;
+pub mod http;
+pub mod metrics;
+mod unmeshed_network;
+
+use unmeshed_network::UnmeshedNetwork;
+
 #[derive(Debug)]
 pub struct Index {
     namespaces: NamespaceIndex,
-    services_by_ip: HashMap<IpAddr, ServiceRef>,
-    service_info: HashMap<ServiceRef, ServiceInfo>,
+    services_by_ip: HashMap<IpAddr, ResourceRef>,
+    service_info: HashMap<ResourceRef, ServiceInfo>,
```

Comment:
in a thread elsewhere, we discuss renaming things to "resource" terminology.

if we are doing that, these fields should also be called `resources_by_ip`, `resource_info`, etc.

Reply:
I don't think `resource_` is useful prefix... Everything in Kubernetes is a resource. What about these resources is meaningful?